packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -829,12 +878,7 @@ export default class PciInstancesAddController {
     )
       .then(() => this.goBack(message, 'success'))
       .catch((err) => {
-        this.CucCloudMessage.error(
-          this.$translate.instant(
-            'pci_projects_project_instances_add_common_error',
-            { message: get(err, 'data.message', '') },
-          ),
-        );
+        this.commonError(err);
```

Comment:
**nitpick:** Please create functions with meaning name.
Convention: verb + action

In this scenario, `handleError` makes more sense.

Reply:
Taken care. Thanks.